docs/examples/eandt.py

```diff
@@ -0,0 +1,70 @@
+from gladier import GladierBaseClient, generate_flow_definition, GladierBaseTool
+from pprint import pprint
+
+
+def makeFiles(**data):
+    import os
+    input_path = data['make_input']
+
+    if '~' in input_path:
+        input_path = os.path.expanduser(input_path)
+
+    with open(input_path+"/file1.txt", "w") as file1:
+        file1.write("This is file no. 1")
+
+    with open(input_path+"/file2.txt", "w") as file2:
+        file2.write("This is file no. 2")
+
+    with open(input_path+"/file3.txt", "w") as file3:
+        file3.write("This is file no. 3")
+
+    return input_path
+
+
+@generate_flow_definition
+class MakeFiles(GladierBaseTool):
+    funcx_functions = [makeFiles]
+    required_input = [
+        'make_input',
+        'funcx_endpoint_compute'
+    ]
+
+
+@generate_flow_definition
+class CustomTransfer(GladierBaseClient):
+    gladier_tools = [
+        MakeFiles,
+        'gladier_tools.posix.Tar',
+        'gladier_tools.posix.Encrypt',
+        'gladier_tools.globus.Transfer',
+    ]
+
+
+if __name__ == '__main__':
+    flow_input = {
+        'input': {
+            # Set this to the folder in which you want to run the makeFiles function in
+            'make_input': '',
```

Comment:
`/tmp/myfiles` could be a safe default to use

Reply:
Not sure if that is true in windows.

Reply:
That's a good point, do you know a better cross-platform alternative?

Reply:
This does require funcx-endpoint being installed on a user's machine, and I don't think Windows is supported. Maybe we could ignore Windows?

Reply:
True.